### DISCUSSION TITLE
v0.20.7

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1560] [Releases]
> [!WARNING]  
> `v0.20.x` has breaking changes and requires action. Read the original release notes here: https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.0

## What's New
* Add Server location to results and export by @svenvg93 in https://github.com/alexjustesen/speedtest-tracker/pull/1542

## What's Changed
* Fix value rounding  by @svenvg93 in https://github.com/alexjustesen/speedtest-tracker/pull/1550
* Laravel 11.11.1 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1552
* Release v0.20.7 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1559

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.20.6...v0.20.7

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.7'>v0.20.7</a>.</em>

**Comment by seh2000** [comment]
Hi Alex,
Updated all OK
Would it possible to add to the Dashboard Server and perhaps Server location in a future release?

**Reply by svenvg93** [reply]
Server location is added in this release. It’s not enabled by default. You can enable it in the results table 😊. Or do you mean something different? 

What do you mean with ”Dashboard Server”?

**Reply by seh2000** [reply]
@svenvg93 yes, I understood it is not enabled by default, I should have been more clear and have said I had enabled under Results.
I for got the "the" between Dashboard Server, I wanted to say:
Is it possible with a release in the future to add to the Dashboard the Server name and the Server location?

**Reply by alexjustesen** [reply]
Better filters and server details are planned for when I rebuild the dashboard in a couple versions.

**Comment by Lropss** [comment]
I am trying to set up telegram bot. As soon as I add .env file with TELEGRAM_BOT_TOKEN="MY TOKEN" I am getting 
this error in logs 
Database conne 188: Database connection [sqlite ] not configured.
![Screenshot 2024-07-22 213151](https://github.com/user-attachments/assets/74d05e18-6136-458d-8dd6-0894e245fc65)

**Reply by alexjustesen** [reply]
If you don't already have a `.env` pass `TELEGRAM_BOT_TOKEN` in your docker-compose environment variables. Adding a `.env` file can cause conflicts which is why you're probably getting that error.

**Reply by Lropss** [reply]
![Screenshot 2024-07-24 123031](https://github.com/user-attachments/assets/834ba3e6-7213-496f-b9a9-98af37487815)
![Screenshot 2024-07-24 123103](https://github.com/user-attachments/assets/3ed96c17-8a53-4e57-afee-764a0fea5cba)
![Screenshot 2024-07-24 123248](https://github.com/user-attachments/assets/324fe3c3-e701-41f9-95bd-a3e656a7b006)
![Screenshot 2024-07-24 123306](https://github.com/user-attachments/assets/42ff1c40-5cfa-48fe-a4a1-0a45a842e1cb)

Bot is not working, what am I doing wrong?
